sample-app/src/main/java/com/rocket/android/core/data/sample/app/MainActivity.kt

```diff
@@ -1,18 +1,21 @@
 package com.rocket.android.core.data.sample.app
 
-import android.app.Activity
 import android.os.Bundle
-import android.util.Log
-import com.rocket.android.core.data.permissions.Permissions
+import androidx.activity.viewModels
+import androidx.fragment.app.FragmentActivity
 import com.rocket.android.core.data.permissions.di.CoreDataProvider
 import com.rocket.android.core.data.sample.app.databinding.ActivityMainBinding
 
-class MainActivity : Activity() {
+class MainActivity : FragmentActivity() {
```

Comment:
WTF put all this into Compose right now!! 😂

Reply:
OMG I should do it in compose 😢 , I will add a Task to the project!

Reply:
#6 issue created!